docs/os-ref.md

```diff
@@ -4,615 +4,357 @@ title: aragonOS reference documentation
 sidebar_label: Reference documentation
 ---
 
-*Documentation for [aragonOS](https://github.com/aragon/aragonOS) v3.1.2*
+aragonOS A framework that enables flexible and upgradeable governance mechanisms by creating and assigning permissions to multiple entities. This doc assumes you have some knowledge about solidity, if not checkout [Solidity](https://solidity.readthedocs.io/) before you jump into AragonOS.
```

Comment:
I feel like, where this file improves a lot the docs about aragonOS, it would require a few changes and deep review from someone like @sohkai or @izqui. So can we maybe open a new PR for this file? Taking it out we can merge everything else immediatly